doc/user/content/connect-sources/cdc-postgres-direct.md

```diff
@@ -108,51 +160,100 @@ As a _superuser_ (`cloudsqlsuperuser`):
 
 {{< /tabs >}}
 
-Once logical replication is enabled:
+## Step 2. Create a publication
 
-1. Grant the required privileges to the replication user:
+Once logical replication is enabled, the next step is to create a publication with the tables that you want to replicate to Materialize. You'll also need create a user for Materialize and give the user permissions to manage replication.
+
+{{< tabs >}}
+{{< tab "AWS RDS">}}
+
+1. For each table that you want to replicate to Materialize, set the [replica identity](https://www.postgresql.org/docs/current/sql-altertable.html#SQL-ALTERTABLE-REPLICA-IDENTITY) to `FULL`:
 
     ```sql
-    ALTER ROLE "dbuser" WITH REPLICATION;
+    ALTER TABLE table1 REPLICA IDENTITY FULL;
     ```
 
-    And ensure that this user has the right permissions on the tables you want to replicate:
+    This setting makes sure sufficient information is written to the replication stream for  `UPDATE` and `DELETE` operations.
+
+    <!-- THE LINKED PAGE DOESN'T TALK ABOUT INCREASED CPU. WE SHOULD FIND ANOTHER PAGE TO LINK TO. COMMENTING THIS OUT IN THE MEANTIME. As a heads-up, you should expect a performance hit in the database from increased CPU usage. For more information, see the [PostgreSQL documentation](https://www.postgresql.org/docs/current/logical-replication-publication.html). -->
+
+1. Create a [publication](https://www.postgresql.org/docs/current/logical-replication-publication.html) with the tables you want to replicate:
+
+    _For specific tables:_
 
     ```sql
-    GRANT CONNECT ON DATABASE dbname TO dbuser;
-    GRANT USAGE ON SCHEMA schema TO dbuser;
-    GRANT SELECT ON ALL TABLES IN SCHEMA schema TO dbuser;
+    CREATE PUBLICATION mz_source FOR TABLE table1, table2;
     ```
 
-    **Note:** `SELECT` privileges on the tables you want to replicate are needed for the initial table sync.
-
-1. Set the replica identity to `FULL` for the tables you want to replicate:
+    _For all tables in the database:_
 
     ```sql
-    ALTER TABLE repl_table REPLICA IDENTITY FULL;
+    CREATE PUBLICATION mz_source FOR ALL TABLES;
     ```
 
-    This setting determines the amount of information that is written to the WAL in `UPDATE` and `DELETE` operations.
+    The `mz_source` publication will contain the set of change events generated from the specified tables, and will later be used to ingest the replication stream.
 
-    As a heads-up, you should expect a performance hit in the database from increased CPU usage. For more information, see the [PostgreSQL documentation](https://www.postgresql.org/docs/current/logical-replication-publication.html).
+    We strongly recommend that you **limit** publications only to the tables you need.
```

Comment:
If you'd like a little context to add: if the publication includes other tables (e.g. they create a publication for all tables), Materialize still ingests the data for the other tables, but then just discards the data as it comes in. For a contrived example, if you only wanted to ingest data for 1 table but the publication contains 1,000 tables, we're wasting a lot of resources filtering out all of the stuff we don't want.